<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UO AnimMUL Analyzer v0.1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="4,2">
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="7" column="4">
         <widget class="QLabel" name="label_frameoffsetdec_val">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>2</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="9" column="6">
         <widget class="QLabel" name="label_height_val">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="9" column="3">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>yCenter:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="4">
         <widget class="QLabel" name="label_xcenter_val">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSpinBox" name="spinBox_idxindex">
          <property name="maximum">
           <number>44000</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_idxindex">
          <property name="text">
           <string>Entry index</string>
          </property>
         </widget>
        </item>
        <item row="9" column="5">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Height:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="9" column="4">
         <widget class="QLabel" name="label_ycenter_val">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="5" column="5">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Frame number:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Lookup (dec):</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3" colspan="4">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Anim*.mul</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLabel" name="label_unknown_val">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Unknown:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Size:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="8" column="1">
         <widget class="QLabel" name="label_size_val">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="label_lookupdec_val">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="7" column="5">
         <widget class="QLabel" name="label_9">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>FrameOffset (h):</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="label_lookuphex_val">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>2</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Lookup (hex):</string>
          </property>
         </widget>
        </item>
        <item row="7" column="6">
         <widget class="QLabel" name="label_frameoffsethex_val">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>2</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="6" column="3" colspan="2">
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>Stream position for the frame:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="5" colspan="2">
         <widget class="QLabel" name="label_frame_streampos_val">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="5" column="4">
         <widget class="QLabel" name="label_framecount_val">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Anim*.idx</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="8" column="6">
         <widget class="QLabel" name="label_width_val">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>FrameCount: </string>
          </property>
         </widget>
        </item>
        <item row="8" column="5">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Width:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="3">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>FrameOffset (d):</string>
          </property>
         </widget>
        </item>
        <item row="5" column="6">
         <widget class="QSpinBox" name="spinBox_framenumber">
          <property name="maximum">
           <number>44000</number>
          </property>
         </widget>
        </item>
        <item row="8" column="3">
         <widget class="QLabel" name="label_xcenter">
          <property name="text">
           <string>xCenter:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="7">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Files Directory:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_browse">
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0" colspan="7">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Anim file number</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_animfilenum">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_26">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Status:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_status_val">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>8</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Idle</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
